### PR TITLE
Reordering layertree groups enhancement

### LINKED ISSUE #1347: Groups reorder enhancement - dashed zone at the bottom
This is a nice-to-have.
When dragging a group to the bottom, the dashed zone is displayed and then disappear forever. It should stay visible.

### PR DESCRIPTION
**Pull request by olivierSemet**
Close #1347

demo: https://oliviersemet.github.io/ngeo/reordering-layers-enhancement/examples/contribs/gmf/apps/desktop/index.html

@pgiraud @fredj

### CODE REVIEW

File: src/directives/sortable.js [hunk 1]
```diff
@@ -96,9 +102,13 @@ ngeo.sortableDirective = function($timeout) {
...
             }
 
+            if (options['dragHoverClass'] !== undefined) {
+              dragHoverClass = options['dragHoverClass'];
+            }
+
             dragListGroup = new goog.fx.DragListGroup();
             dragListGroup.addDragList(element[0],
-                goog.fx.DragListDirection.DOWN);
+                goog.fx.DragListDirection.DOWN, true, dragHoverClass);
```
Comment (pgiraud) [inline src/directives/sortable.js]: Any example of a usage of the `dragHoverClass` options? Looks like it's not used at all.
I'd rather add it if we really need it.

**Comment by pgiraud** [comment]
We shouldn't do this dynamically in javascript. We should be able to do this in pure css. The dragList element should take all the available height.
I'll work on this one.

File: src/directives/sortable.js (hunk 1 above)
Comment (olivierSemet) [inline src/directives/sortable.js]: No I used it in a former solution. The class name passed to through this option is added to the list when the user start to drag and till the end of the drag event. My first solution was to increase the height of the list with an additional CSS class. I though that this parameter might be useful in the future.

**Comment by pgiraud** [comment]
Superseded by #1375.

**Pull request closed without merging**